addons/repair/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from psycopg2 import sql
+from openupgradelib import openupgrade
+
+
+def update_repair_fee_invoice_relation(env):
+    column = "invoice_line_id"
+    openupgrade.rename_columns(
+        env.cr, {"repair_fee": [(column, None)]}
+    )
```

Comment:
You call it in all function, why not move args in variable and call once?
Easier to see what was renamed.
<pre>
 _rename_columns = {
    "repair_fee": [('invoice_line_id', None)],
    "repair_line": [('invoice_line_id', None)],
    "repair_order": [('invoice_id', None)],
}
....
@openupgrade.migrate()
    def migrate(env, version):
        openupgrade.load_data(
            env.cr, "repair",
            "migrations/13.0.1.0/noupdate_changes.xml"
        )
        openupgrade.rename_columns(env.cr, _rename_columns)
        update_repair_fee_invoice_relation(env)
        update_repair_line_invoice_relation(env)
        update_repair_order_invoice_relation(env)
        update_repair_order_user_id(env)
</pre>